### DISCUSSION TITLE
Changing collumn number

### DISCUSSION THREAD

**Discussion by ayyilmaz** [Dem13n/discussion-cards#8] [Ideas]
Currently, Number of primary cards show as two collumn . I think adding an option for changing collunm numbers on grid layout will increase functionality of the extension. We canchange collumns between 2 or 3 or one block. May be this option is added to admin panel  to force show the same layout for averone or added as javascript code to change layout at front-end side. Increasing collumn number provides to see more discussions at the same time. 2 collumns is good but may be 3 or 4 collums also good for reach more discussion momently.

**Comment by Dem13n** [comment]
I will do it
